recipes/parso/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "parso" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5815f3fe254e5665f3c5d6f54f086c2502035cb631a91341591b5a564203cffb" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - parso
+    - parso.pgen2
+    - parso.python
+
+about:
+  home: https://github.com/davidhalter/parso
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: A Python Parser
+  description: "###################################################################\nparso - A Python Parser\n###################################################################\n\n.. image:: https://secure.travis-ci.org/davidhalter/parso.png?branch=master\n\
```

Comment:
Maybe shorten the `description` to just:

> Parso is a Python parser that supports error recovery and round-trip parsing for different Python versions (in multiple Python versions). Parso is also able to list multiple syntax errors in your python file.

rather than copy-pasting the entire README